src/sentry/testutils/factories.py

```diff
@@ -1014,6 +1017,10 @@ def create_group(project, **kwargs):
         if "metadata" in kwargs:
             metadata = kwargs.pop("metadata")
             kwargs["data"].setdefault("metadata", {}).update(metadata)
+        if "status" not in kwargs:
+            kwargs["status"] = GroupStatus.UNRESOLVED
+            kwargs["substatus"] = GroupSubStatus.NEW
+
```

Comment:
This fixes any "incorrect substatus for unresolved group" errors that stem from stricter group status validation